database-connection/login.js

```diff
@@ -1,4 +1,9 @@
 function login(email, password, callback) {
+
+  if (!email || !password) {
```

Comment:
```suggestion
  if (typeof email !== 'string' || typeof password !== 'string') {
```
`!` check is misleading as it accepts non-string types as valid values.

Reply:
Resolving -
![image](https://user-images.githubusercontent.com/44966979/133986626-ee01674b-25aa-4a62-a3ba-59bf8bfbb4e3.png)